lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -339,7 +339,7 @@ svi_autostate:
   kind: boolean
   get_value: '/^(?:no )?autostate$/'
   set_value: "<state> autostate"
-  default_value: true
+  default_value: false
```

Comment:
This seems suspect as a global change - are you sure it's not a platform difference that needs to be accounted for?